### ISSUE TITLE
Working directory name containing '[' fail to open

### ISSUE THREAD

**Issue by Silver-Fullbuster** [donmaiq/Mpv-Playlistmanager#32]
Log: 
```
[playlistmanager] cd : Cannot find path 'D:\P2P\Downloaded\[Shir] Sankarea [Blu-Ray][720p][Hi10][Duall-Audio]\' because it does not
[playlistmanager] exist.
[playlistmanager] At line:6 char:11
[playlistmanager] +           cd "D:\P2P\Downloaded\[Shir] Sankarea [Blu-Ray][720p][Hi10] ...
[playlistmanager] +           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[playlistmanager]     + CategoryInfo          : ObjectNotFound: (D:\P2P\Download...][Duall-Audio]\:String) [Set-Location], ItemNotFoundE
[playlistmanager]    xception
[playlistmanager]     + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.SetLocationCommand
[playlistmanager]
```

Removing all instances of '[' in the directory name resolves the issue

**Comment by jonniek** [comment]
Can you try this fix 080b337f739855041b936d3c2a10c6f6c9270d4f

**Comment by Silver-Fullbuster** [comment]
That fixes it! Thanks for the super quick response!
